prog/weaver/main.go

```diff
@@ -474,10 +476,17 @@ func main() {
 		muxRouter.Methods("GET").Path("/metrics").Handler(metricsHandler(router, allocator, ns, dnsserver))
 		statusMux := http.NewServeMux()
 		statusMux.Handle("/", muxRouter)
-		Log.Println("Listening for metrics requests on", statusAddr)
+		Log.Println("Listening for status+metrics requests on", statusAddr)
```

Comment:
Simplification: as you have introduced `-metrics-addr`, we could serve only the status on `-status-addr`. However, it is non-backward compatible change.

Reply:
Yes, I think it's better to get this change in to remove the attach vector and then spend more time thinking what interfaces we really want to expose.  I'm not aware `--status-addr` was ever used to fetch the status.

Reply:
ack!